stdlib/effect.mli

```diff
@@ -19,10 +19,18 @@
 type _ t = ..
 (** The type of effects. *)
 
+exception Unhandled_effect : 'a t -> exn
+(** Exception raised when an effect is performed and there is no matching
+    handler for that effect. The argument is the unhandled effect. *)
+
+exception Continuation_already_taken
+(** Exception raised when a continuation is resumed with a [continue] or
+    [discontinue] but the continuation has already been resumed. *)
+
```

Comment:
Did we agree that the exception should be `Effect.Unhandled` instead of `Effect.Unhandled_effect`, or are we still on the fence?

Also, @dbuenzli's comments on the doc strings in #11419 are still relevant and should be addressed.